from braintree.exceptions.unexpected_error import UnexpectedError

class TimeoutError(UnexpectedError):
    pass
